### PR TITLE
WINC-511: Use RuntimeClass in e2e tests

### PR DESCRIPTION
**Pull request by mburke5678**
Documents:  [WINC-511: Use RuntimeClass in e2e tests](https://github.com/openshift/windows-machine-config-operator/pull/2040)  

Update Windows runtime and deployment for 2019 and 2012

4.12+

QE review:
- [x] QE has approved this change.

### CODE REVIEW

**Comment by mburke5678** [comment]
@sebsoto @rrasouli  PTAL

File: modules/creating-runtimeclass.adoc [hunk 1]
```diff
@@ -23,15 +23,21 @@ scheduling:
...
     kubernetes.io/os: 'windows'
     kubernetes.io/arch: 'amd64'
-    node.kubernetes.io/windows-build: '10.0.17763'
-  tolerations: <3>
-  - effect: NoSchedule
-    key: os
-    operator: Equal
-    value: "Windows"
+    node.kubernetes.io/windows-build: '10.0.17763' <3>
+  tolerations: <4>
```
Comment (sebsoto) [inline modules/creating-runtimeclass.adoc:27]: I dont see the footnote for <4>

**Review by sebsoto — COMMENTED** [review]
@mburke5678 LGTM besides from the one comment

**Comment by kalexand-rh** [comment]
The `branch/enterprise-4.16` label has been added to this PR.

This is because your PR targets the `main` branch and is labeled for enterprise-4.15. And any PR going into `main` must also target the latest version branch (enterprise-4.16).

If the update in your PR does NOT apply to version 4.16 onward, please re-target this PR to go directly into the appropriate version branch or branches (enterprise-4.x) instead of main.

File: modules/sample-windows-workload-deployment.adoc [hunk 2]
```diff
@@ -51,29 +51,26 @@ spec:
...
-        name: windows
+        name: "windows"
+      runtimeClassName: windows2019 <3>
 ----
-
-[NOTE]
-====
-When using the `mcr.microsoft.com/powershell:<tag>` container image, you must define the command as `pwsh.exe`. If you are using the `mcr.microsoft.com/windows/servercore:<tag>` container image, you must define the command as `powershell.exe`. For more information, see Microsoft's documentation.
-====
+<1> Specify the container image to use, either `mcr.microsoft.com/powershell:<tag>` or `mcr.microsoft.com/windows/servercore:<tag>`. The container image must match the Windows version running on the node.
```
Comment (skrthomas) [inline modules/sample-windows-workload-deployment.adoc:70]:
```suggestion
<1> Specify the container image to use: `mcr.microsoft.com/powershell:<tag>` or `mcr.microsoft.com/windows/servercore:<tag>`. The container image must match the Windows version running on the node.
```

**Review by skrthomas — COMMENTED** [review]
One nit. LGTM otherwise :)

**Pull request merged**